src/Bart/Diesel.php

```diff
@@ -125,6 +172,37 @@ public static function disableDefault()
 	{
 		self::$allowDefaults = false;
 	}
+
+
+	private static function checkTestScribe($className, $arguments)
+	{
+
+		/** @noinspection PhpUndefinedNamespaceInspection */
+		/** @noinspection PhpUndefinedClassInspection */
+		if (self::isTestGeneratorRun()
+			&& \Box\TestScribe\App::$shouldInjectMockObjects
+		) {
+			// This code block is to support PHPUnit test generator.
+			// @see https://github.com/box/TestScribe
+			// The class 'Box\TestScribe\App' should only be loaded
+			// when this method is executed by the test generator.
+			// It allows the test generator to substitute the real class instance with a
+			// mock object controlled by the test generator so that the test
+			// generator can monitor the creation and execution of the class
+			// being resolved.
+			/** @noinspection PhpUndefinedNamespaceInspection */
+			$mock = \Box\TestScribe\App::createMockedInstance(
+				$className,
+				$arguments
+			);
+			if ($mock) {
```

Comment:
This is an inconsistent return type. You should return null or an instance of something, but not void. Alternatively, Bart could start using PhpOption...